sdk/tables/Azure.Data.Tables/src/TablesClientOptions.cs

```diff
@@ -9,24 +9,24 @@ namespace Azure.Data.Tables
     /// <summary>
     /// Options to configure the requests to the Table service.
     /// </summary>
-    public class TableClientOptions : ClientOptions
+    public class TablesClientOptions : ClientOptions
```

Comment:
![](https://media4.giphy.com/media/2UEMMgwPi5nx8gNyPo/giphy.gif)

Reply:
Subtle. But I can see the reason.

Reply:
The reason is that this options class is shared across `TableServiceClient` and `TableClient`